src/main/resources/swagger/api-docs.yaml

```diff
@@ -2037,7 +2177,7 @@ paths:
             {
               "type": "array",
               "items": {
-                "type": "string"
+                $ref: '#/definitions/BillingProjectMembership'
```

Comment:
The change to `BillingProjectMembership ` doesn't seem to be reflected in the actual response. For example, the response model schema is shown as:
```
[
  {
    "projectName": "string",
    "role": "string"
  }
]
```
However, the actual response is just an array of strings:
``
[
  "broad-dsde-alpha",
  "broad-dsde-dev"
]
```